lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -133,6 +134,15 @@ import qualified Data.ByteArray as BA
 import qualified Data.ByteString as BS
 import qualified Data.ByteString.Lazy as BL
 
+
+-- maximum number of inputs in a valid transaction
+maxNumberOfInputs :: Int
+maxNumberOfInputs = 255
+
+-- maximum number of outputs in a valid transaction
+maxNumberOfOutputs :: Int
+maxNumberOfOutputs = 255
```

Comment:
:+1:
I think however that there are a few places in this very module which also use these constants in a hard-coded form. It'd be good if we could replace them too by using their named equivalent newly created :smile:

Reply:
found two more places where 255 is used